virtualargofleet/virtualargofleet.py

```diff
@@ -346,9 +366,10 @@ def get_an_output_filename(output_folder):
 
         if output:
             # Close the ParticleFile object (used to exporting and then deleting the temporary npy files in older versions)
-            log.info("starting ParticleFile export/close/clean from %s" % opts['output_file'].fname)
+            # fname not available in older versions of parcels
+            log.info("starting ParticleFile export/close/clean")
```

Comment:
Is there a specific reason for removing the explicit log mention of ``opts['output_file'].fname`` ?

Reply:
`ParticleFile.fname` is not available on previous version of Parcels.